docs/workflows/sync.rst

```diff
@@ -0,0 +1,106 @@
+Synchronize a Repository
+========================
+
+Users can populate their repositories with content from an external source like PyPI by syncing
+their repository.
+
+Create a Repository
+-------------------
+
+Start by creating a new repository named "foo"::
+
+    $ http POST http://localhost:8000/pulp/api/v3/repositories/ name=foo
```

Comment:
for the sake of consistency, why not use $PULP_BASE (or whatever the variable name will end up being).